internal/gps/constraints.go

```diff
@@ -393,3 +393,19 @@ func (m ProjectConstraints) override(pr ProjectRoot, pp ProjectProperties) worki
 
 	return wc
 }
+
+// IsSemverRange is a helper that returns whether the given constraint is a range constraint.
+// branches, tags, revisions, and explicit versions should return false.
+func IsSemverRange(c Constraint) bool {
```

Comment:
Can we change this to `IsSpecificSemVer` instead? And change `!isSimpleVersion` to `isSimpleVersion`?

Reply:
Do you mean so `IsSpecificSemVer` returns `true` for branches/tags/revisions/specific versions?

Reply:
Yes.